app/views/guides/_form.html.erb

```diff
@@ -72,8 +72,29 @@
       <div class='form-group'>
         <%= f.submit "Save Draft", class: 'btn btn-success', name: :save_draft %>
         <%= link_to "Preview", document_preview_url(guide), class: 'btn btn-default' %>
-        <%= f.submit "Publish", class: 'btn btn-success', name: :publish %>
+        <% if guide.latest_edition.present? && guide.latest_edition.approved? %>
+          <%= f.submit "Publish", class: 'btn btn-success', name: :publish %>
+        <% end %>
       </div>
     </fieldset>
   <% end %>
+
+  <% if guide.latest_edition.present? %>
+    <% if guide.latest_edition.persisted? && !guide.latest_edition.review_requested? %>
```

Comment:
What happens if someone requests a review and then creates another edition? I think users only need to request a review once per-draft. What do you think?